R/pkg/R/mllib.R

```diff
@@ -99,10 +115,7 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
             return(new("GeneralizedLinearRegressionModel", jobj = jobj))
           })
 
-#' Fits a generalized linear model (R-compliant).
-#'
-#' Fits a generalized linear model, similarly to R's glm().
-#'
+#' @title Fit a generalized linear model
```

Comment:
The original title is okay. Shall we change this to `Generalized Linear Models (R-compliant)`?